src/Haskell/Ide/Engine/Hoogle.hs

```diff
@@ -21,29 +21,6 @@ import           System.Environment
 import           Text.HTML.TagSoup
 import           Text.HTML.TagSoup.Tree
 
--- ---------------------------------------------------------------------
-
-hoogleDescriptor :: PluginId -> PluginDescriptor
-hoogleDescriptor plId = PluginDescriptor
-  { pluginId = plId
-  , pluginName = "hoogle"
-  , pluginDesc =
-         "Hoogle is a Haskell API search engine, which allows you to search "
-      <> "many standard Haskell libraries by either function name, or by approximate "
-      <> "type signature. "
-  , pluginCommands =
-      [ PluginCommand "info" "Look up the documentation for an identifier in the hoogle database" infoCmd
-      , PluginCommand "lookup" "Search the hoogle database with a string" lookupCmd
-      ]
-  , pluginCodeActionProvider = Nothing
-  , pluginDiagnosticProvider = Nothing
-  , pluginHoverProvider = Nothing
-  , pluginSymbolProvider = Nothing
-  , pluginFormattingProvider = Nothing
-  }
-
```

Comment:
Where has this moved to? why?

Reply:
The Hoogle plugin didn't have any providers, and the `lookup` and `info` commands were just used as a part of haddock when it provides documentation (which are now just called directly instead of via the reactor). If the plugin doesn't provide any providers, it becomes a no-op to HIE, so I thought it would make sense to remove it as a plugin and make it more of a utility module for Hoogle related functions. i.e. if the user disabled the Hoogle plugin, nothing would happen: haddock still calls the methods here regardless.